alfalfa_worker/Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM michaelwetter/ubuntu-1804_jmodelica_trunk AS base
+FROM ubuntu:18.04 AS base
```

Comment:
nice! Maybe 22.04 will fix the issues you are seeing? Let's wait until it is released next month!